### DISCUSSION TITLE
Add a book-list with recommended books

### DISCUSSION THREAD

**Discussion by curateideas** [microsoft/alguidelines#152] [Ideas]
It would be very helpful to have a book-list with the recommended books as sources of information! Also tag them to show for which topic they are useful.

**Comment by waldo1001** [comment]
Great idea!
Let's use this discussion to list the books first, before we add them to a page on the site.  Anyone?

Here's a start:

[Automated Testing in Microsoft Dynamics 365 Business Central - Second Edition](https://www.packtpub.com/product/automated-testing-in-microsoft-dynamics-365-business-central-second-edition/9781801816427)

**Reply by curateideas** [reply]
That one was the first that came to my mind as well ;-)

**Reply by christianbraeunlich** [reply]
because we all read that recently 😜

**Comment by christianbraeunlich** [comment]
KB's: https://github.com/mynavblog/ALForBeginners

**Comment by curateideas** [comment]
Programming Microsoft Dynamics 365 Business Central - Sixth Edition
https://www.packtpub.com/product/programming-microsoft-dynamics-365-business-central-sixth-edition/9781789137798


Your First 20 Hours with Business Central
https://sparebrained.com/your-first-20-hours-with-microsoft-dynamics-365-business-central/


Implementing Azure DevOps Solutions
https://www.packtpub.com/product/implementing-azure-devops-solutions/9781789619690

**Comment by christianbraeunlich** [comment]
Copied from https://bit.ly/bcresources and blanked the already mentioned:

- [Dynamics 365 Business Central Field Guide](https://www.hougaard.com/book/)

- ~[Your First 20 Hours with Business Central](https://sparebrained.com/your-first-20-hours-with-microsoft-dynamics-365-business-central/)~

- ~[Programming Microsoft Dynamics 365 Business Central - Sixth Edition](https://www.packtpub.com/application-development/programming-microsoft-dynamics-365-business-central-sixth-edition)~

- [Dynamics 365 Business Central Development Quick Start Guide](https://www.packtpub.com/business/dynamics-365-business-central-development-quick-start-guide)

- [Dynamics 365 Business Central Cookbook](https://www.packtpub.com/business-other/microsoft-dynamics-365-business-central-cookbook)

- [Mastering Microsoft Dynamics 365 Business Central](https://www.packtpub.com/application-development/mastering-microsoft-dynamics-365-business-central)

- [Implementing Microsoft Dynamics 365 Business Central On-Premise](https://www.packtpub.com/application-development/implementing-microsoft-dynamics-365-business-central-premise-fourth-edition)

I haven't read these books yet, but since they made it on the list, I decided to list them anyway.
